src/main/kotlin/racingcar/RacingCars.kt

```diff
@@ -3,11 +3,14 @@ package racingcar
 import number.Numbers.makeRandomsToRacing
 
 class RacingCars(private var cars: Cars) {
-    fun racing(attempts: Int): List<Int> {
+    fun racing(attempts: Int): List<List<Int>> {
+        val allRoundPositions = ArrayList<List<Int>>()
```

Comment:
`mutableListOf`를 활용해보는 건 어떨까요?

```suggestion
        val allRoundPositions = mutableListOf<List<Int>>()
```

Reply:
피드백 반영하였습니다.! 👋